### PR TITLE
FFM-7341: Updates release notes for GO 0.1.7 sdk, corrects java version num

### PR DESCRIPTION
**Pull request by StrawberryPad**

### CODE REVIEW

**Review by conormurray95 — DISMISSED** [review]

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Feature Flags
-date: 2023-03-08T10:00
+date: 2023-03-30T10:00
```
Comment (roshnisarangadharan) [inline release-notes/feature-flags.md:3]:
```suggestion
date: 2023-03-31T10:00
```

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -37,6 +37,29 @@ The Java server SDK has been updated to version **1.2.2**, with the following up
 
 * Minor internal changes were made to make it easier for developers to use classes that were previously marked private.
 
+## March 29, 2023
```
[the remark this answers is not in the document]
Comment (roshnisarangadharan) [reply release-notes/feature-flags.md:40]: Do you add version details for FF release notes?

File: release-notes/feature-flags.md [hunk 3]
```diff
@@ -37,6 +37,29 @@ The Java server SDK has been updated to version **1.2.2**, with the following up
...
+
+This release does not include early access features.
+
+### Fixed issues
+
+#### Feature Flags SDKs
+
+The **Go server SDK** has been updated to version **0.1.7**, with the following updates.
+
+* Previously if a custom logger was passed in to the SDK through the harness.WithLogger(logger) function, the custom logger was not used when logging HTTP requests. This could cause HTTP request logs to be in a different format, and appear to be logged at an `error` level instead of correctly logged as `debug`. This has been resolved, and all logs emitted by the SDK now go through the custom logger if it's passed in. (FFM-7327)
```
Comment (roshnisarangadharan) [inline release-notes/feature-flags.md:56]: Should this be `harness.WithLogger(logger)`?

Comment (roshnisarangadharan) [inline release-notes/feature-flags.md:56]: ....to be logged at an `error` level instead of `debug`.

**Review by roshnisarangadharan — APPROVED** [review]

File: release-notes/feature-flags.md (hunk 3 above)
Comment (StrawberryPad) [reply release-notes/feature-flags.md:56]: good question, i've been leaving programmatic functions as is, but maybe they should be in <code> -- we can add to style discussion

Comment (StrawberryPad) [reply release-notes/feature-flags.md]: done

**Pull request merged**